app/views/videos/index.html.haml

```diff
@@ -3,7 +3,7 @@
     %header
       = link_to category_path(category) do
         %h3= category.name.upcase
-      .videos.row
+      .video.row
```

Comment:
Not sure why you changed this. It should match up to the format in the ui view https://github.com/eaglerockdude/myflix/blob/master/app/views/ui/home.html.haml#L1-L8

I suggest comparing the two and making sure what you have matches it.